shoppingcart/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM python:3.8.0-slim
 
 COPY ./dist /dist
-RUN pip install /dist/cloudstate-0.5.0-py3-none-any.whl
+RUN pip install /dist/cloudstate-0.1.0-py3-none-any.whl
```

Comment:
One question, after we publish the package, could that change? Or will we continue to use shoppingcart in our build?

Reply:
What do you mean with change? this would be the shopping cart example/sample, no?

Reply:
About what we talked about before, having a shoppingcart regardless of the build. Using official packages instead of compiled in python-support's own build.

Reply:
Ah right; and we wanted to have the shopping cart independent from the distribution of the library.